test/tests/selftest/pbs_default_timeout.py

```diff
@@ -0,0 +1,104 @@
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.selftest import *
+from ptl.utils.plugins.ptl_test_runner import TimeOut
+
+
+class TestDefaultTimeout(TestSelf):
+    """
+    Test suite to verify increase default testcase timeout working properly.
+    """
+
+    def test_default_timeout(self):
+        """
+        Verify that test not timedout after 180 sec.
+        i.e. after previous default timeout value
+        """
+        try:
+            mssg = 'sleeping for %ssec and minimum-testcase-timeout is %ssec' \
+                   % (MINIMUM_TESTCASE_TIMEOUT/2, MINIMUM_TESTCASE_TIMEOUT)
+            self.logger.info(mssg)
+            time.sleep(MINIMUM_TESTCASE_TIMEOUT/2)
+        except TimeOut, e:
+            mssg = 'Timed out after 600 second'
+            self.assertEqual(mssg, e.message)
+
+    def test_timeout_greater_default_value(self):
+        """
+        Verify that test timedout after 600 sec.
+        """
+        try:
+            mssg = 'sleeping for %ssec and minimum-testcase-timeout is %ssec' \
+                   % (MINIMUM_TESTCASE_TIMEOUT+1, MINIMUM_TESTCASE_TIMEOUT)
+            self.logger.info(mssg)
+            time.sleep(MINIMUM_TESTCASE_TIMEOUT+1)
+        except TimeOut, e:
+            mssg = 'Timed out after 600 second'
+            self.assertEqual(mssg, e.message)
+        else:
+            self.fail('Test is not timeout after minimum timeout period')
+
+    @timeout(200)
+    def test_timeout_decorator_less_default_value(self):
+        """
+        If timeout decorator value is less than 600 then
+        default testcase timeout is considered
+        """
+        try:
+            mssg = 'sleeping for %ssec and minimum-testcase-timeout is %ssec' \
+                   % (MINIMUM_TESTCASE_TIMEOUT/2, MINIMUM_TESTCASE_TIMEOUT)
+            self.logger.info(mssg)
+            time.sleep(MINIMUM_TESTCASE_TIMEOUT/2)
+        except TimeOut, e:
+            mssg = 'Timed out after 600 second'
+            self.assertEqual(mssg, e.message)
```

Comment:
Please add an error message to assert and do this for the rest of the code as well.